test/integration/instance/restore.test.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const chai = require('chai');
+
+const expect = chai.expect;
+const Support = require('../support');
+const { DataTypes } = require('@sequelize/core');
+const sinon = require('sinon');
+
+const current = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('Instance'), () => {
```

Comment:
We have update conventions for our new test suites but we only require them to be applied on new files:

- Use TypeScript
- No need for 'use strict'
- Use import instead of require
- Name the sequelize instance "sequelize" instead of "current"
- Don't use `this`. Replace `this.sequelize` with sequelize. If you need to expose data from before & beforeEach, use `beforeAll2` and `beforeEach2`